tests/backup/backup_helper.go

```diff
@@ -289,7 +289,7 @@ func CreateBackup(backupName string, clusterName string, bLocation string, bLoca
 		return err
 	}
 
-	err = backupSuccessCheck(backupName, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx)
+	err = backupSuccessCheck(backupName, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second, ctx, true)
```

Comment:
we can modify to accept the parameter from the testcase and pass it down to backupSucessCheck